pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -487,6 +541,13 @@ public boolean commitSegment(String rawTableName, final String committingSegment
     return true;
   }
 
+  private int getRealtimeTableFlushSizeForTable(String tableName) {
```

Comment:
Please rename tableName as rawTableName, thanks.
Also, make this method protected, so we can use it in test to return the value we want to return.